department-of-veterans-affairs/abd-vro
# Feature - update Contention Classification Lookup Table to v0.3

## What was the problem?

 - Expand to additional diagnostic codes to increase the amount of coverage of VA.gov claims that can be classified by the contention classifier endpoint
 - Fixes issue where one of the codes was missed when parsing .csv

Associated tickets or Slack threads:
- https://app.zenhub.com/workspaces/contention-classification-640a24db5dcd08457cebbc79/issues/gh/department-of-veterans-affairs/vagov-claim-classification/209

## How does this fix it?[^1]


## How to test this PR
- Run `pytest` per [README](https://github.com/department-of-veterans-affairs/abd-vro/tree/develop/domain-cc#unit-tests).
- You can also run the code (`./gradlew :domain-cc:dockerComposeUp`) and make POST requests against `/classifier`


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
